tailwind.config.js

```diff
@@ -48,6 +48,17 @@ module.exports = {
       l: '32px',
       xl: '48px',
       xxl: '64px'
+    },
+    borderWidth: {
+      1: '1px',
+      2: '2px',
+      3: '3px',
+      4: '4px'
+    },
+    borderRadius: {
```

Comment:
I have noticed a few things here:

- DEFAULT -> we do not use all uppercase namings, please rename it to "default"
- There is also a border radius with "16px" that is being used for modals/popups [reference](https://www.figma.com/file/nsXR2h0KwciWpuwKRD58FX/Mumble?type=design&node-id=521-3177&mode=design&t=HW5IGMiLDVAlFNZG-0)

Reply:
Sorry, regarding my first point -> since there is kind of a smaller and larger border radius, you could name it with "s" and "m". 

Also, do not forget to update borders.mdx with the 16px border radius example

Reply:
@malinovic fixed

added also a large variant (only used on hover for profile avatar - on screen 'empfohlene user'.
added another border with (6px - user avatar circle)

Reply:
Great observation 👍🏻